test/math/SafeMath.test.js

```diff
@@ -88,4 +88,21 @@ contract('SafeMath', () => {
       await assertJump(this.safeMath.div(a, b));
     });
   });
+
+  describe('mod', function () {
+    it('modulo correctly', async function () {
```

Comment:
Could you add a couple more test cases? Since modulo is a bit of a tricky operation, I think it'd be nice to test when `a < b`, `a == b`, `a > b`, and `a == b * k` (for some integer `k`).